Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1339,6 +1367,11 @@ extension BrowserViewController : Transitionable {
             }
         }
         self.homePanelController?.view.hidden = false
+        if options.toView === self {
+            startTrackingAccessibilityStatus()
+        } else {
+            stopTrackingAccessibilityStatus()
+        }
```

Comment:
I wish our tab handling code wasn't so fragile; having to spread out these tracking calls everywhere makes me nervous that we'll regress something later. Maybe it'd be useful to add a programmatic override for `UIAccessibilityIsVoiceOverRunning` so that we can create some simple KIF tests.